templates/accessRule.interface.hbs

```diff
@@ -0,0 +1,10 @@
+<tr data-rule-index="{{index}}">
+    <td class="align-middle">{{{allow}}}</td>
```

Comment:
I can't remember whether we agreed on this, but I remember preferring to align this to the top when there are multiple lines. I understand a lot (not necessarily most) of scenarios would only use one line, which you mentioned. Can you add a screenshot for what it looks like when it's top aligned?